udata/api/commands.py

```diff
@@ -59,3 +63,23 @@ def validate():
         success('API specifications are valid')
     except schemas.SchemaValidationError as e:
         exit_with_error('API specifications are not valid', e)
+
+
+@grp.command()
+@click.option('-u', '--user-email', help='User\'s email')
+@click.option('--uri', default='http://localhost:8080/login', help='Client\'s redirect uri')
+def create_oauth_client(user_email, uri):
+    '''Creates an OAuth2Client instance in DB'''
+    user = User.objects(email=user_email).first()
+    if user is None:
+        exit_with_error('No matching user to email')
+
+    client = OAuth2Client.objects.create(
+        name='test-client',
+        owner=user,
+        redirect_uris=[uri]
```

Comment:
suggestion to enable adding several uris in the cli instead of just one (but not sure about chosing `+`as main separator) : 
```        redirect_uris=uri.split('+')```